beacon-chain/blockchain/process_block.go

```diff
@@ -110,6 +112,39 @@ func (s *Service) onBlock(ctx context.Context, signed *ethpb.SignedBeaconBlock,
 		return err
 	}
 
+	// If slasher is configured, forward the attestations in the block via
+	// an event feed for processing.
+	if featureconfig.Get().EnableSlasher {
+		// Feed the indexed attestation to slasher if enabled. This action
+		// is done in the background to avoid adding more load to this critical code path.
+		go func() {
+			signedHeader, err := blockutil.SignedBeaconBlockHeaderFromBlock(signed)
```

Comment:
You are double importing block header here. We already do it in `validate_beacon_blocks.go`.

Are you thinking about initial syncing? if yes, a better place will be in `round_robin.go`